en/news/_posts/2017-07-21-ruby-prize-2017.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: news_post
+title: "Nominations now being accepted for Ruby Prize 2017"
+author: "Ruby Association"
+translator:
+date: 2017-07-21 00:00:00 +0000
+lang: en
+---
+
+We are very pleased to announce you that Ruby Prize will be held in this year 2017!
```

Comment:
```
held in this year 2017
```

I recommend using `held in 2017` or `held this year` which makes it a bit more clear.